### PR TITLE
v2.0.3

### PR DESCRIPTION
**Pull request by geemus**
Dropped changelog from gemspec as it doesn't appear to exist and building gem fails when specified files are absent.

### CODE REVIEW

**Comment by geemus** [comment]
Once this is merged I can build/release.

**Review by dosas — APPROVED** [review]

**Comment by geemus** [comment]
Since this was blocked by a reviewer with commit, I (at least for now) gave myself permission for direct pushes. This is inline with other fog repos, and unblocked. Having done that I directly pushed and released, so this version should now be out and available.

**Comment by dosas** [comment]
> Since this was blocked by a reviewer with commit, I (at least for now) gave myself permission for direct pushes. This is inline with other fog repos, and unblocked. Having done that I directly pushed and released, so this version should now be out and available.

Thank you! https://rubygems.org/gems/fog-ovirt/versions/2.0.3

**Pull request closed without merging**
